ground/gcs/src/plugins/config/cfg_vehicletypes/configfixedwingwidget.cpp

```diff
@@ -66,19 +66,19 @@ void ConfigFixedWingWidget::setupUI(SystemSettings::AirframeTypeOptions frameTyp
 
     if (frameType == SystemSettings::AIRFRAMETYPE_FIXEDWING) {
         // Setup the UI
-        setComboCurrentIndex(m_aircraft->aircraftType, m_aircraft->aircraftType->findText("Fixed Wing"));
-        setComboCurrentIndex(m_aircraft->fixedWingType, m_aircraft->fixedWingType->findText("Elevator aileron rudder"));
-        m_aircraft->fwRudder1ChannelBox->setEnabled(true);
+//        setComboCurrentIndex(m_aircraft->aircraftType, m_aircraft->aircraftType->findText("Fixed Wing"));
```

Comment:
This kind of stuff is commented out because it's out of scope for multirotors, but definitely in scope for the next round of revisions once my direction is approved/revamped.